examples/source/interactivity-dynamic-content/Dynamic_Accordion.html

```diff
@@ -0,0 +1,171 @@
+<!---
+
+author: elisameyer
+tags
+  - dynamic-content
+
+--->
+<!--
+  ## Introduction
+
+  This code snippet shows how you can dynamically update data in `amp-accordion `sections using `amp-bind` and `amp-list`.ß
+-->
+<!DOCTYPE html>
+<html ⚡="">
+  <head>
+    <title>Dynamic Accordion</title>
+    <meta charset="utf-8" />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <link rel="canonical" href="<% canonical %>">
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+    <!-- ## Setup -->
+    <!-- Import the `amp-accordion` component to display content in accordion -->
+    <script
+      async
+      custom-element="amp-accordion"
+      src="https://cdn.ampproject.org/v0/amp-accordion-0.1.js"
+    ></script>
+    <!-- Import the `amp-bind` component to update accordion state based on user input -->
+    <script
+      async
+      custom-element="amp-bind"
+      src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"
+    ></script>
+    <!-- Import the `amp-list` component to dynamically display the names of hot or cold countries -->
+    <script
+      async
+      custom-element="amp-list"
+      src="https://cdn.ampproject.org/v0/amp-list-0.1.js"
+    ></script>
+    <!-- Import the `amp-mustache` component to dynamically display items from amp-list -->
+    <script
+      async
+      custom-template="amp-mustache"
+      src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"
+    ></script>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+    <style amp-custom="">
+      .show {
+        display: block;
+      }
+      .hide {
+        display: none;
+      }
+    </style>
+  </head>
+  <body>
+      <h2>
+        Select which climate you prefer
+      </h2>
+      <!-- ## Implementation -->
+      <!--
```

Comment:
Same thing as with the other samples, make sure to add a comment before every relevant element.

Reply:
updated (added extra comment for accordion section)